Representor.xcodeproj/project.pbxproj

```diff
@@ -12,6 +12,9 @@
 		276A2C051A7BA4EE004BCC6F /* BlueprintTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 276A2C031A7BA4EE004BCC6F /* BlueprintTests.swift */; };
 		276A2C061A7BA4EE004BCC6F /* BlueprintTransitionTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 276A2C041A7BA4EE004BCC6F /* BlueprintTransitionTests.swift */; };
 		276A2C081A7BA500004BCC6F /* blueprint.json in Resources */ = {isa = PBXBuildFile; fileRef = 276A2C071A7BA500004BCC6F /* blueprint.json */; };
+		276A2C291A7BC0CB004BCC6F /* URITemplate.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 276A2C1E1A7BC07C004BCC6F /* URITemplate.framework */; };
```

Comment:
I wonder if there is a way to make GitHub ignoring these files (not .gitignore of course)